index.bs

```diff
@@ -558,6 +563,52 @@ spec: SHA2; urlPrefix: http://csrc.nist.gov/publications/fips/fips180-4/fips-180
 
   <!-- ####################################################################### -->
 
+  # Authoring Considerations # {#authoring-considerations}
+
+  <em>This section is not normative.</em>
+
+  ## Versioned resources ## {#versioned-resources}
+
+  <a>Integrity metadata</a> is only useful if the resource can be reasonably
+  guaranteed to remain the same. It is therefore important that if resources
+  are updated they receive a new URL. Typically this might take the form of a
+  version number as part of the path or file name, which would ensure that a
+  resource would only be expected to change as a result of a compromised
+  server.
+
+  <div class="example">
+  <pre>
+    &lt;script src="https://example.com/v1.0/example-framework.js"
+            integrity="sha384-Li9vy3DqF8tnTXuiaAJuML3ky+er10rcgNR/VqsVpcw+ThHmYcwiB1pbOxEbzJr7"
+            crossorigin="anonymous"&gt;&lt;/script&gt;
+  </pre>
+  </div>
+
+  ## Coping with errors ## {#coping-with-errors}
+
+  When an integrity violation is detected, the resource will not parse or
+  execute. This could cause problems for users, so the system should be
+  designed to cope with errors.
+
+  If an alerting or logging system exists, the integrity error should be
+  sent to that so that the violation can be resolved safely.
```

Comment:
I'm not sure what "resolved safely" means here. What do you expect developers to do with `error` events? I agree that it's useful to collect them to ensure that users are getting a good experience on your sites, but it's not clear to me what action you're recommending here.

Reply:
I’m fleshing this section out a bit with an example as requested in your other point, but I’ve expanded this paragraph to read:
>  If an alerting or logging system exists, the integrity error should also
  be sent there so that appropriate steps can be taken. If a single resource
  on a CDN is seen to have been compromised, then general usage of that
  system should be evaluated for other possible problems. For example, if
  other resources without integrity metadata have also been loaded from the
  same CDN, the impact of their potential compromise should be evaluated.

I’ll push that in the same commit as the fallback example if you’re happy with that.

Reply:
Maybe a lighter tone s/compromised/modified/?

Reply:
We use ”compromise” in other parts of the spec, but no problem with being less worrisome. ”Modified” works.